### DISCUSSION TITLE
Bridge

### DISCUSSION THREAD

**Discussion by DEVANNA75** [blockscout/blockscout#9272] [Support] [answered]
Salve,ho effettuato un bridge si MANTA PACIFIC 5 giorni fa, c'è stata la transazione su Smart contact ma l'indirizzo di ricezione non è il mio

**Comment by GimluCom** [comment, marked as answer]
Hi. This is the Blockscout Github. We cannot help you with this issue, instead please contact the blockchain's team directly.

> Ciao. Questo è il Github di Blockscout. Non possiamo aiutarti con questo problema, contatta direttamente il team della blockchain.

**Comment by mesaifulop** [comment]
I'm bridge Rari ETH to Arbitruim ETH 1 week ago 
But it's not completed
I'm not received my wallet 
They tell us 1 week. But 1 week gone my funds not received
Transection I'd:  0x0d863b9b50c3f14b09920871e5b68fa56394a3f806514612fcb975908109d489

My wallet address:  0x847DDB001a5530F865aa5Cae1eAeE6763619fc90

Please help 🙏
